src/components/component-playground.jsx

```diff
@@ -109,6 +113,7 @@ module.exports = React.createClass({
     preview: function() {
       var params = {
         component: this.constructor.getSelectedComponentClass(this.props),
+        ref: ((ref) => this.refs.preview = ref),
```

Comment:
It should, but it doesn't. The reason is `loadChild` sets the ref as a string ref, and not a callback ref. Then, anything rendered in the context of `Provider` (as its child) doesn't save any string refs, it only does so with callbacks.
I have yet to understand why, but React docs say [string refs are legacy](https://facebook.github.io/react/docs/more-about-refs.html#the-ref-string-attribute), maybe it has something to do with that.